plugins/woocommerce/src/Admin/PluginsInstallLoggers/AsynPluginsInstallLogger.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\PluginsInstallLoggers;
+
+/**
+ * A logger to log plugin installation progress in real time to an option.
+ */
+class AsynPluginsInstallLogger implements PluginsInstallLogger {
```

Comment:
It seems the change got replaced with a rebase?

```suggestion
class AsyncPluginsInstallLogger implements PluginsInstallLogger {
```